### ISSUE TITLE
Provide the example as a loadable project like the flutter gallery 

### ISSUE THREAD

**Issue by ride4sun** [ArcticZeroo/flutter-search-bar#6]
I think is just to much work consuming a small example like this. Better would be to open example sub-folder which would include the complete example as a flutter project

**Comment by ArcticZeroo** [comment]
Agreed, feel free to provide it, otherwise I'll tackle this and any other open issues when I have some free time
